app_feup/pubspec.yaml

```diff
@@ -39,9 +39,12 @@ dependencies:
   flutter_markdown: ^0.6.0
   intl: ^0.17.0
   crypto: ^3.0.1
+  add_2_calendar: ^1.4.0
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.2
+  material_design_icons_flutter: 5.0.6595
```

Comment:
Do we need to depend directly on this version or can we depend on any version bigger than 5.0.6595?
Such as ^5.0.6595